includes/specials/SpecialProperties.php

```diff
@@ -66,7 +66,13 @@ protected function getGroupName() {
 	}
 
 	private function getLimitOffset() {
-		return $this->getRequest()->getLimitOffset();
+		$request = $this->getRequest();
+		if ( method_exists( $request, 'getLimitOffsetForUser' ) ) {
+			// MW 1.35+
+			return $request->getLimitOffsetForUser( $this->getUser() );
+		} else {
+			return $request->getLimitOffset();
+		}
```

Comment:
You can write this without the else part:

```php
		if ( method_exists( $request, 'getLimitOffsetForUser' ) ) {
			// MW 1.35+
			return $request->getLimitOffsetForUser( $this->getUser() );
		}

		return $request->getLimitOffset();
```

Reply:
True, but I was using the same code block that I was using elsewhere where it wasn't returning, but rather setting a variable, so I kept the else block

Reply:
That is part of the reason why using a guard clause (the form in my variation) is nice.

Reply:
https://www.entropywins.wtf/blog/2019/01/14/readable-functions-guard-clause/